driver.go

```diff
@@ -721,19 +800,19 @@ func (d *Driver) getKey() (*hcloud.SSHKey, error) {
 	return stype, nil
 }
 
-func (d *Driver) getRemoteKeyWithSameFingerprint(pubkey_byte []byte) (*hcloud.SSHKey, error) {
-	pubkey, _, _, _, err := ssh.ParseAuthorizedKey(pubkey_byte)
+func (d *Driver) getRemoteKeyWithSameFingerprint(publicKeyBytes []byte) (*hcloud.SSHKey, error) {
+	publicKey, _, _, _, err := ssh.ParseAuthorizedKey(publicKeyBytes)
 	if err != nil {
 		return nil, errors.Wrap(err, "could not parse ssh public key")
 	}
 
-	fp := ssh.FingerprintLegacyMD5(pubkey)
+	fp := ssh.FingerprintLegacyMD5(publicKey)
 
-	remotekey, _, err := d.getClient().SSHKey.GetByFingerprint(context.Background(), fp)
+	remoteKey, _, err := d.getClient().SSHKey.GetByFingerprint(context.Background(), fp)
 	if err != nil {
-		return remotekey, errors.Wrap(err, "could not get sshkey by fingerprint")
```

Comment:
not related to this PR, but might be worth sharing: Since [a few Go releases](https://blog.golang.org/go1.13-errors), Go can wrap errors now using the %w flag. Example: `fmt.Errorf("could not do xzy: %w")`